packages/analyticsdx-template-lint/src/schemas/layout-schema.json

```diff
@@ -534,7 +547,7 @@
       ]
     },
     "visibility": {
-      "description": "Controls if this item is visible. Value must be 'Disabled' (valid only for Variable items), 'Hidden', 'Visible', or a {{...}} expression against variables that evaluates to one of those or true (Visible) or false (Hidden).",
+      "description": "Controls if this item is visible. Value must be 'Disabled' (valid only for Variable items and tiles), 'Hidden', 'Visible', or a {{...}} expression against variables that evaluates to one of those or true (Visible) or false (Hidden).",
```

Comment:
@mblumreich - These `description`s in this schema file will show as hover text in vscode.